tests/cpp_unit_tests/test_fault.cpp

```diff
@@ -50,21 +53,20 @@ TEST_CASE("Test fault") {
 
     SUBCASE("Test get_short_circuit_output sym") {
         ComplexValue<true> const i_f_pu = 1.0 + 1.0i;
-        ComplexValue<true> const i_f = i_f_pu * base_i;
 
-        FaultShortCircuitOutput<true> output = fault.get_short_circuit_output<true>(i_f_pu, u_rated);
+        FaultShortCircuitOutput output = fault.get_short_circuit_output<true>(i_f_pu, u_rated);
         CHECK(output.id == 1);
         CHECK(output.energized);
-        CHECK(output.i_f == doctest::Approx(cabs(i_f)));
-        CHECK(output.i_f_angle == doctest::Approx(0.25 * pi));
+
+        // TODO(NITISH) sym sc output case
```

Comment:
```suggestion
        // TODO add fault test post implementation
```

The output functions to modify are only of components other than fault. The fault output function would depend on how its implemented. Lets do this later then.